sdk/schemaregistry/schema-registry/src/conversions.ts

```diff
@@ -1,12 +1,12 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { SchemaProperties, Schema } from "./models";
+import { Schema, SchemaProperties } from "./models";
 
 import {
   SchemaGetByIdResponse,
-  SchemaRegisterResponse,
   SchemaQueryIdByContentResponse as SchemaQueryIdByDefinitionResponse,
+  SchemaRegisterResponse,
```

Comment:
the changes here seem to be formatting ones, are they necessary? Jonathan did formatting work recently here so I do not expect to see more formatting changes.

Reply:
Same comment applies to some other modules as well.

Reply:
All I did was `rushx format`. 😔

Reply:
Perhaps trying fetching the tip of upstream main and merge it here? I think that will resolve this issue.